pydoctor/epydoc2stan.py

```diff
@@ -474,9 +475,11 @@ def resolve_types(self) -> None:
     def format(self) -> Tag:
         r: List[Tag] = []
 
-        r += format_desc_list('Parameters', self.parameter_descs)
-        if self.return_desc:
-            r += format_desc_list('Returns', [self.return_desc])
+        # Only include parameters and returns if any are documented
+        if any(p.body for p in self.parameter_descs) or (self.return_desc and self.return_desc.body):
```

Comment:
If I read correctly, if only the returns value is documented, all parameters will still display. Maybe 'Parameters' and 'Returns' should be independently shown ? Or maybe it makes send to keep it like this?

Reply:
This was how I understood the comment at https://github.com/twisted/pydoctor/issues/473#issuecomment-1139160974. I can make them independent of each other though, no problem.

Reply:
Yeah, I'm not sure what's the current behaviour for such cases without annotations actually.

Reply:
Made these checks/sections independent.